test/fixtures/responses/index.js

```diff
@@ -1,5 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+function moreThan400Items() {
+  let items = [];
+  const oneItem = require('./get-trustlines-more-than-400-items.json');
+  for (let i = 0; i < 401; i++) {
+    items.push(oneItem);
+  }
+  return items;
```

Comment:
In the words of James Brown, you can make it funky(tional) as follows:
```
function moreThan400Items(item) {
  return Array[401].fill(item);
}
```
then call it with the item to inject as an arg so it's generic, or require it as you see fit.